src/sentry/api/bases/organization.py

```diff
@@ -285,7 +294,8 @@ def convert_args(self, request, organization_slug, *args, **kwargs):
         except Organization.DoesNotExist:
             raise ResourceDoesNotExist
 
-        self.check_object_permissions(request, organization)
+        with sentry_sdk.start_span(op="check_object_permissions_on_organization"):
```

Comment:
I expect this to be mostly redundant to the `get_project_access_in_teams` span in practice, but I thought it was important to show which DB queries are part of this span and that `get_project_access_in_teams` is taking up most of the time relative to them.